client/src/pages/Lobby.js

```diff
@@ -188,6 +188,8 @@ const Lobby = (props) => {
         </Snackbar>
       )}
     </>
+  ) : (
+    <Interview userData={userData} />
```

Comment:
Yeah I don't love this... makes it hard to read and maintain, rather find a way to make Interview contained separately in a different route

Reply:
Fixed